tests/testthat/test-carehomes-support.R

```diff
@@ -495,28 +503,31 @@ test_that("Can control the seeding", {
 
   initial_y <- mod$transform_variables(initial$state)
 
-  expect_equal(initial_y$N_tot3, sum(p$N_tot))
-  expect_equal(initial_y$N_tot2, sum(p$N_tot))
+  expect_equal(initial_y$N_tot_sero_2, sum(p$N_tot))
```

Comment:
sero_1?

Reply:
Corrected, and also reordered for clarity